### DISCUSSION TITLE
Couldn't load Web UI translation for selected locale (en). - qBittorrent 4.5.5 & v1.7.3

### DISCUSSION THREAD

**Discussion by MaksimKozub** [VueTorrent/VueTorrent#1092] [Q&A] [answered]
Hi all
I'm trying to configure qBittorrent 4.5.5 & v1.7.3 but it doesn't want to work, log says  
Using custom Web UI. Location: "app/vuetorrent".
(W) 2023-09-07T14:54:48 - Couldn't load Web UI translation for selected locale (en).

Also the message "Couldn't load Web UI translation for selected locale (en)." appears when I open Web interface, I was trying to check previous versions of VueTorrent - the problem is the same

Please help to find the problem 
**UPD**
I'm using docker, just added in the container:
Host/volume -> /DATA/AppData/qbittorrent/config/vuetorrent
Path in container -> /vuetorrent
and it works!

**Comment by Larsluph** [comment]
Every log you see in the logs view is coming from qBittorrent itself.

The only relevant logs for VueTorrent are inside your browser's console.

Everything else can be ignored safely.

**Reply by MaksimKozub** [reply]
Hey, thanks
Here is what I have when open browser's console
http://xxx.xxx.x.xxx:8080/ -> 500 Internal Server Error ->  Unacceptable file type, only regular file is allowed.
http://xxx.xxx.x.xxx:8080/favicon.ico ->  500 Internal Server Error ->  Unacceptable file type, only regular file is allowed.

**Reply by Larsluph** [reply, marked as answer]
That means that qBittorrent's HTTP server is unable to access VueTorrent's files.

Make sure that the public folder is present at the specified location for the WebUI and that the paths are correct (if you're using docker, make sure that the path is the docker one and not the host one)

**Reply by MaksimKozub** [reply]
thanks, I'm using docker, just added in the container:
Host/volume -> /DATA/AppData/qbittorrent/config/vuetorrent
Path in container ->  /vuetorrent

and it works!
